homeassistant/components/tado_v1.py

```diff
@@ -0,0 +1,131 @@
+"""
+Support for the (unofficial) tado api.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/tado_v1/
+"""
+
+import logging
+import urllib
+
+from datetime import timedelta
+import voluptuous as vol
+
+from homeassistant.helpers.discovery import load_platform
+from homeassistant.helpers import config_validation as cv
+from homeassistant.const import (
+    CONF_USERNAME, CONF_PASSWORD)
+from homeassistant.util import Throttle
+
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = 'tado_v1'
+
+REQUIREMENTS = ['https://github.com/wmalgadey/PyTado/archive/'
+                '0.1.10.zip#'
+                'PyTado==0.1.10']
+
+TADO_V1_COMPONENTS = [
+    'sensor', 'climate'
+]
+
+CONFIG_SCHEMA = vol.Schema({
+    DOMAIN: vol.Schema({
+        vol.Required(CONF_USERNAME, default=''): cv.string,
+        vol.Required(CONF_PASSWORD, default=''): cv.string
+    })
+}, extra=vol.ALLOW_EXTRA)
+
+MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=10)
+
+CONST_TADO_DATA = 'tado_v1_data'
```

Comment:
Let's change the prefix from `CONST_` to `DATA_` to match existing conventions. Maybe just `DATA_TADO`.